wagtail_streamfield_migration_toolkit/test/tests/test_streamchangedetector.py

```diff
@@ -165,3 +176,27 @@ def test_nested_struct_child_renamed(self, mock_class):
         ):
```

Comment:
We should use `self.subTest` when running multiple tests in a test method like this, it can give better output for failures